snooty/test_postprocess_granular.py

```diff
@@ -0,0 +1,51 @@
+"""An alternative and more granular approach to writing postprocessing tests.
+   Eventually most postprocessor tests should probably be moved into this format."""
```

Comment:
Let's make _this_ file `test_postprocess.py` and call the old one something more curmudgeonly to encourage new work here!

Reply:
Good call!! Updated!